Packs/BmcHelixRemedyForce/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "BmcHelixRemedyForce",
+    "description": "Integration of BmcHelixRemedyForce with Demisto",
```

Comment:
Add a better description to the pack and change `Demisto` to `Cortex XSOAR`.

Reply:
Yes we would add proper description for the same and also change to Cortex XSOAR.

Reply:
Description added.